src/main/java/com/alienfast/bamboozled/ruby/config/RubyBuildConfigurationPlugin.java

```diff
@@ -85,6 +86,11 @@ public boolean isApplicableTo( Plan plan ) {
     }
 
     @Override
+    public boolean isApplicableTo(@NotNull ImmutablePlan plan) {
+        return false;
```

Comment:
I don't know, you need to trace the hierarchy and see if it meets the interface.  I'm no expert with bamboo, I just happened to know the atlassian plugin ecosystem.